README.md

```diff
@@ -7,6 +7,16 @@ This is the developer documentation. For user documentation, please consult http
 
 ## Building
 
+Make sure you have all dependencies installed for the Aerospike C client and asbackup.
+See https://github.com/aerospike/aerospike-client-c#build-prerequisites for more C client information.
+Below are dependencies for asbackup only.
+- openssl (1.1 or 3)
+- An event library: libuv, libevent, or libev
```

Comment:
maybe mention that this is the same event lib uses to compile the c-client. It is also optional

Reply:
added a note